src/remembrall/classes/Fields.py

```diff
@@ -72,3 +72,16 @@ def __init__(self, value: str):
 
     def __str__(self):
         return datetime.strftime(self.value, format)
+    
+class Address(Field):
+    def __init__(self, value: str):
+        self.value = value.strip()
+
+    def edit_address(self, new_address: str):
+        Address.check_address(new_address)
```

Comment:
добав так, надійніше
```suggestion
        Address.check_address(new_address.strip())
```

Reply:
Добре, дякую, буде зроблено.